pyproject.toml

```diff
@@ -17,9 +17,11 @@ flake8 = "^4.0.1"
 flake8-bugbear = "^22.3.23"
 pytest = "^7.1.1"
 pytest-cov = "^3.0.0"
-types-python-slugify = "^5.0.4"
 types-python-dateutil = "^2.8.16"
 
+[tool.poetry.group.dev.dependencies]
+types-python-slugify = "^8.0.0.3"
+
```

Comment:
We should harmonise the way we define `group` with poetry.  
Example, we have the poetry old fashion way (<1.2):
 `[tool.poetry.dev-dependencies]`
 And the new way (poetry >=1.2) with the added `group`:
 `[tool.poetry.group.dev-dependencies]`
 
 https://python-poetry.org/blog/announcing-poetry-1.2.0/#dependency-groups

Reply:
I have consolidated all dev dependencies under tool.poetry.group.dev.dependencies.